{{ .ProjectSnake }}/.aspect/cli/package-json-scripts.star

```diff
@@ -0,0 +1,34 @@
+aspect.register_rule_kind("js_binary", {
+    "From": "@aspect_rules_js//js:defs.bzl",
+})
+
+def declare_targets(ctx):
+    for file in ctx.sources:
+        if not file.query_results["scripts"][0]:
+            continue
+        for (script, cmd) in file.query_results["scripts"][0].items():
+            # Heuristics to try to understand the structure of a random npm script
+            cmd_parts = cmd.split(" ")
+
+            # Looks like it just directly runs an entry point script
+            if cmd_parts[0] == "node" and len(cmd_parts) == 2:
+                ctx.targets.add(
+                    name = script,
+                    kind = "js_binary",
+                    attrs = {
+                        "entry_point": cmd_parts[-1],
```

Comment:
You could potentially add `"deps": [aspect.Label(name=":node_modules")]` 🤷

Reply:
Actually that's broken:

```
2024/09/10 14:59:15 Failed to read starlark value Label{repo: "", pkg: "", name: "//:node_modules"}
panic: Failed to read starlark value Label{repo: "", pkg: "", name: "//:node_modules"}

goroutine 1 [running]:
log.Panicf({0x101b2adb3?, 0x10?}, {0x14000e40a98?, 0x100c8afec?, 0x14000e40b18?})
        GOROOT/src/log/log.go:439 +0x64
github.com/aspect-build/silo/cli/core/gazelle/common/starlark/utils.ReadRecurse({0x102b3bc00?, 0x140007495f0?}, 0x60?)
        cli/core/gazelle/common/starlark/utils/util.go:151 +0x1ec
```